casper/src/main/scala/coop/rchain/casper/util/comm/DeployService.scala

```diff
@@ -49,8 +49,8 @@ class GrpcDeployService(host: String, port: Int) extends DeployService[Task] {
     response.toProtoString
   }
 
-  def addBlock(b: BlockMessage): Task[Unit] =
-    Task.delay {
-      blockingStub.addBlock(b)
-    }.void
+  def addBlock(b: BlockMessage): Task[(Boolean, String)] = Task.delay {
+    val response = blockingStub.addBlock(b)
```

Comment:
@birchmd Should this not return `Either[InvalidBlock, Unit]` though?

Reply:
No, the `blockingStub.addBlock` is calling this: https://github.com/rchain/rchain/pull/1176/files#diff-72121b5dc081b75121497cdaf951797dR70